### PR TITLE
Develop term of use

### PR DESCRIPTION
**Pull request by Weltgeist**
-Added term of use page and some content from a external source.
-Used handlebar instead of react.
-Couldn't find Luis static page so, I included it separately.

Any suggestion, on how to include it the react way is welcomed.

### CODE REVIEW

File: client/src/components/Forum/StaticPage.js [hunk 1]
```diff
@@ -9,7 +10,20 @@ export default function StaticPage() {
   return (
     <MyBox>
     <div>
-        This is a static page
+      <main>
+        <article id="term-of-use">
+          <section id="sub-section" class="container">
+          { termOfUse.map( (term,idx) => (
+            <article id="ownership">
```
Comment (luciancic) [inline client/src/components/Forum/StaticPage.js]: The `id` attribute can come from `term.name` and it needs a `key` attribute that you can set with `idx`. See [react lists and keys on why](https://reactjs.org/docs/lists-and-keys.html) on why.

**Review by luciancic — CHANGES_REQUESTED** [review]
Congrats on your first PR! Nicely done. Just a few details to iron out before I can merge. Could you rename `StaticPage.js` to `TermsOfUse.js` and the imports, links and route. Also see my comment on the `article` attributes. If u have questions about updating the PR and the review process you know where to find me. Thanks!

File: client/src/components/Forum/StaticPage.js (hunk 1 above)
Comment (Weltgeist) [reply client/src/components/Forum/StaticPage.js]: I did, thx. 
term:
590b630
id:
d021fa1
I also made some refactoring, added another functional react element

Comment (Weltgeist) [reply client/src/components/Forum/StaticPage.js]: term:
590b630 
id:
d021fa1

**Comment by luciancic** [comment]
Delete terms-of-use.html
Style the link and the content using basic materialUI components
Replace terms of use with privacy policy and code of conduct (separate page) content.

**Comment by luciancic** [comment]
Closing because this seems to be going off on a tangent and a term of use is overkill on an app of this scope.

**Pull request closed without merging**